shared/src/commonTest/kotlin/co/touchlab/kampkit/BreedModelTest.kt

```diff
@@ -116,9 +162,23 @@ class BreedModelTest : BaseTest() {
         assertNotNull(model.getBreedsFromNetwork(0L))
     }
 
+    @ExperimentalTime
     @AfterTest
     fun breakdown() = runTest {
         dbHelper.deleteAll()
+        // The in-memory SQLite native doesn't reset the autoincrement.
+        // We need to destroy it and recreate it, but
+        // it's not possible drop a table in SQLDelight:
+        // https://github.com/cashapp/sqldelight/issues/1870
+        /*        
+        testDbConnection().execute(
+            null,
+            """
+                DELETE FROM sqlite_sequence WHERE name = 'Breed';
+            """.trimIndent(),
+            0
+        )
+        */
```

Comment:
From hacking around on another project this weekend, here's how you can make this work:
1. Hold onto the `SqlDriver` object returned by `testDbConnection()`
2. Call `close()` on it here
That should completely reset the database state when the next test starts.

Reply:
That works!